service/src/test/java/com/epam/esm/service/TagServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package com.epam.esm.service;
+
+import com.epam.esm.config.MapperConfig;
+import com.epam.esm.convert.Convert;
+import com.epam.esm.dto.TagDto;
+import com.epam.esm.entity.Tag;
+import com.epam.esm.repository.TagDao;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.context.web.WebAppConfiguration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@ExtendWith(SpringExtension.class)
+@WebAppConfiguration
+@ContextConfiguration(classes = {ServiceTestConfig.class, MapperConfig.class})
+class TagServiceTest {
+
+    @Autowired
+    TagService tagService;
+
+    @Autowired
+    Convert<Tag, TagDto> convert;
+
+    @Autowired
+    TagDao tagDao;
+
+    @Test
+    void getTags() {
+        List<TagDto> expList = new ArrayList<>();
```

Comment:
it is always a good idea to have a little delimeters, e.g. 'given/when/then' in test code